src/SimulationService.jl

```diff
@@ -245,6 +242,9 @@ end
 # For JSON3 read/write
 StructTypes.StructType(::Type{DataServiceModel}) = StructTypes.Mutable()
 
+# PIRACY to fix JSON3.read(str, DataServiceModel)
+JSON3.Object(x::AbstractDict) = JSON3.read(JSON3.write(x))
```

Comment:
eek, do we have an upstream issue for this?

Reply:
This is because a `JSON3.Object` lazily interprets raw (`Vector{UInt8}`) data.  It's an `AbstractDict`, but it doesn't act like one in a few ways.

Reply:
I want to keep some of the fields of `OperationRequest`/`DataServiceModel` a `JSON3.Object` because its even more immutable than `ImmutableDict`

Reply:
Is JSON3 calling this or are we calling this. If the former, why can't JSON3 just create an appropriate reference?

Reply:
JSON3 is calling it.  

We need to do `JSON3.read(str, DataServiceModel)` where one of the fields of a `DataServiceModel` is `obj::JSON3.Object`.  Apparently providing a `T` in `JSON3.read(str, T)` will change any internal `JSON3.Object` into a `Dict`, so we were hitting a method error for `JSON3.Object(::Dict)`.

I'll make an in JSON3 if there isn't already.